jx_sqlite/query_table.py

```diff
@@ -288,7 +288,12 @@ def query(self, query):
                     data = Data()
                     for c in index_to_columns.values():
                         if c.push_child == ".":
-                            data[c.push_name] = c.pull(result.data[0])
+                            if not data[c.push_name]:
```

Comment:
maybe use `data[c.push_name] += [c.pull(result.data[0])]` which will do the null handling. Unfortunately, it creates a singleton list.